extension/dhcp/dhcp.js

```diff
@@ -89,6 +89,42 @@ async function dhcpDiscover(intf) {
   });  
 }
 
+async function dhcpServerStatus(serverIp) {
+  let result = false;
+  let cmd = util.format('sudo nmap -sU -p 67 --script=dhcp-discover %s -oX - | %s', serverIp, xml2jsonBinary);
+  log.info("Running command:", cmd);
+  try {
+    const cmdresult = await execAsync(cmd);
+    let output = JSON.parse(cmdresult.stdout);
+    let kvs = output.nmaprun && output.nmaprun.host
```

Comment:
consider using lodash.get() for readability
https://lodash.com/docs/4.17.15#get